src/index.js

```diff
@@ -0,0 +1,34 @@
+import './style.scss';
+import {
+  AddToDoList,
+  setTasks,
+  showTasks,
+  GetToDoList,
+  updateLocalStorage,
+} from './modules/AddToDoItems.js';
+import { clearAllCompletedTasks } from './modules/Completed.js';
+
+const newTodolistInput = document.querySelector('#add-to-do > input');
```

Comment:
- [ ] At the moment, I am not able to add todo items to the list because this selector does not find any dom node. Please ensure you are selecting the right node.
- [ ] Also, this made it impossible to test the remove and edit functionalities. Please ensure all is working fine before submitting for another review.